stockflux-securities-master/src/components/InputForm.js

```diff
@@ -27,47 +37,60 @@ const InputForm = ({ match }) => {
 
   useEffect(() => {
     if (match.params.securityId) {
-      setIsLoading(true);
+      setContextOrState(stateEnum.loading);
       getSecurity(match.params.securityId)
         .then(security => {
-          setIsLoading(false);
-          setErrorMessage(null);
+          setContextOrState(null);
```

Comment:
Don't forget to empty the messages here if there are any:

    setMessages([])

Reply:
Okay sure